widgetastic_manageiq/__init__.py

```diff
@@ -2144,7 +2338,8 @@ class BaseEntitiesView(View):
     """
     @property
     def entity_class(self):
-        return BaseEntity
+        return VersionPick({Version.lowest(): NonJSBaseEntity,
```

Comment:
Can you format this like the other `VersionPick` calls, with the first dictionary item on a newline?